src/docker.test.ts

```diff
@@ -27,6 +27,13 @@ const assertCalledInOrder = (...mocks: jest.Mock[]): void => {
 
```

Comment:
Commit message: "test workflow" --> "Test workflow" since it's a proper noun. I just spot checked a few of the Save Cache jobs, and I think it was more like 6-10 minutes previously, but I didn't investigate comprehensively. Remember to update the pull request description.

Reply:
"test workflow" --> "Test workflow"